ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -113,14 +115,15 @@ def _closest_waypoint_index(self):
                     self.current_pose.position.y)
         
         if self.current_waypoint_ahead is None:
-            possible_waypoint_indices = self._get_waypoint_indices(0, LOOKAHEAD_WPS)
+            possible_waypoint_indices = self._get_waypoint_indices(0,
+                                                                   self.len_base_waypoints)
```

Comment:
There might be a possible race condition if the waypoints_cb() function is called while the waypoints are updated and published.

Reply:
yes.. that crossed my mind as well. I will address that in a future PR 👍